lib/shared/errors.js

```diff
@@ -227,16 +233,20 @@ exports.getDescription = (error) => {
     return codeDescription;
   }
 
```

Comment:
Might be neater if we just have a single:
```js
if (options.userFriendlyDescriptionsOnly) {
  return '';
}
```
here, as I think that'd then allow us to remove both the `!options.userFriendlyDescriptionsOnly` tests further below?